Feature/delete costume approval board member/costume approval board

hyeneung opened the pull request:
## #️⃣연관된 이슈
- this closes #48 

## 📝작업 내용
- 출연자 의상 컨펌 게시글 삭제
- API 테스트 완료

hyeneung committed:
feature: 출연자 의상 컨펌 게시글 삭제 기능 정의 및 구현

hyeneung committed:
feature: controller 작성

pangseon reviewed (changes requested):
현재 삭제에 대한 로직도 특정 부분을 받아서 삭제해야하는데 관련 내용이 작성되지 않았습니다
다른 도메인에서 사용되고 있는 삭제 메서드 참고해서 작업해주세요

hyeneung committed:
feature: exception 코드 추가

hyeneung committed:
feature: 출연자, 업체 의상 승인 게시글 삭제 기능. 게시글 id 통해 삭제하도록 변경

출연자의 삭제 요청과 업체의 삭제 요청을 API 상으로 구분할지 논의 필요

hyeneung committed:
style: annotation 추가

shin0112 reviewed (commented):
수고하셨습니다 👍

pangseon reviewed (approved):
성공

hyeneung committed:
style: merge conflict 해결

Merge remote-tracking branch 'origin/develop' into feature/delete-costume-approval-board-member/costume-approval-board
# Conflicts:
#	src/main/java/com/example/extra/domain/costumeapprovalboard/CostumeApprovalBoard/CostumeApprovalBoardController.java
#	src/main/java/com/example/extra/domain/costumeapprovalboard/entity/CostumeApprovalBoard.java
#	src/main/java/com/example/extra/domain/costumeapprovalboard/exception/CostumeApprovalBoardErrorCode.java
#	src/main/java/com/example/extra/domain/costumeapprovalboard/repository/CostumeApprovalBoardRepository.java
#	src/main/java/com/example/extra/domain/costumeapprovalboard/service/CostumeApprovalBoardService.java
#	src/main/java/com/example/extra/domain/costumeapprovalboard/service/impl/CostumeApprovalBoardServiceImpl.java
